template/bin/create-user

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
it might be worth calling this same as the function name. Also, since its in the path, `create-user` has higher collision chance than `create-user-from-directory-owner`